src/player.cpp

```diff
@@ -11168,7 +11199,11 @@ bool player::has_weapon() const
 
 m_size player::get_size() const
 {
-    return MS_MEDIUM;
+    if ( move_mode == "sneak" ) {
+        return MS_SMALL;
```

Comment:
This doesn't look good: small size makes one harder to hit, including in melee.
Would be better if you just special cased this in sight-related functions.

Reply:
Hm, if it is harder for us to hit smaller targets why not making it harder for them to hit us in this case? 
I looked into this and think that penalizing your dodge roll (easy to implement) and the players melee damage/force/strength (can't find a proper place?) while sneaking is a better and more realistic (you're not as mobile and can't put your full body weight into the attack) option for this?

Reply:
I love the small size effect because it could create options for small door (House with dog door... and zdog inside; cramped maintenance tunnels in sewers; small cave passages; etc)